<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>{{ cookiecutter.tool_name }}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy></property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="hLayout_Main">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect></property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="action_Print_Preview"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Font"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Select_All"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Full_Screen"/>
    <addaction name="action_Wrap_Text"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
    <addaction name="action_Bug"/>
    <addaction name="action_Updates"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="baseSize">
    <size>
     <width>800</width>
     <height>50</height>
    </size></property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string></property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Print"/>
   <addaction name="action_Print_Preview"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
   <addaction name="action_Help"/>
  </widget>
  <action name="action_Open">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/open.png</normaloff>:/main_ui/open.png</iconset></property>
   <property name="text"><string>&amp;Open</string></property>
   <property name="toolTip"><string>Open a file</string></property>
   <property name="shortcut"><string>Ctrl+O</string></property>
  </action>
  <action name="action_Save">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/save.png</normaloff>:/main_ui/save.png</iconset></property>
   <property name="text"><string>&amp;Save</string></property>
   <property name="toolTip"><string>Save the file</string></property>
   <property name="shortcut"><string>Ctrl+S</string></property>
  </action>
  <action name="action_About">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/about.png</normaloff>:/main_ui/about.png</iconset></property>
   <property name="text"><string>&amp;About</string></property>
   <property name="toolTip"><string>About this program</string></property>
  </action>
  <action name="action_Close">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/close.png</normaloff>:/main_ui/close.png</iconset></property>
   <property name="text"><string>&amp;Close</string></property>
   <property name="toolTip"><string>Close the file</string></property>
   <property name="shortcut"><string>Ctrl+W</string></property>
  </action>
  <action name="action_Save_As">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/rename.png</normaloff>:/main_ui/rename.png</iconset></property>
   <property name="text"><string>Save&amp;As</string></property>
   <property name="toolTip"><string>Save the file under a new name</string></property>
   <property name="shortcut"><string>F12</string></property>
  </action>
  <action name="action_Help">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/help.png</normaloff>:/main_ui/help.png</iconset></property>
   <property name="text"><string>&amp;Help</string></property>
   <property name="toolTip"><string>View the documentation</string></property>
   <property name="shortcut"><string>F1</string></property>
  </action>
  <action name="action_Print">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/print.png</normaloff>:/main_ui/print.png</iconset></property>
   <property name="text"><string>&amp;Print</string></property>
   <property name="shortcut"><string>Ctrl+P</string></property>
  </action>
  <action name="action_Print_Preview">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/preview.png</normaloff>:/main_ui/preview.png</iconset></property>
   <property name="text"><string>Print Pre&amp;view</string></property>
   <property name="shortcut"><string>Ctrl+Shift+P</string></property>
  </action>
  <action name="action_New">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/new.png</normaloff>:/main_ui/new.png</iconset></property>
   <property name="text"><string>New &amp;File</string></property>
   <property name="toolTip"><string>Create a new file</string></property>
   <property name="shortcut"><string>Alt+J</string></property>
  </action>
  <action name="action_Exit">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/exit.png</normaloff>:/main_ui/exit.png</iconset></property>
   <property name="text"><string>E&amp;xit</string></property>
   <property name="toolTip"><string>Exit the program</string></property>
   <property name="shortcut"><string>Ctrl+Q</string></property>
  </action>
  <action name="action_Undo">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/undo.png</normaloff>:/main_ui/undo.png</iconset></property>
   <property name="text"><string>Undo</string></property>
   <property name="shortcut"><string>Ctrl+Z</string></property>
  </action>
  <action name="action_Redo">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/redo.png</normaloff>:/main_ui/redo.png</iconset></property>
   <property name="text"><string>Redo</string></property>
   <property name="shortcut"><string>Ctrl+Y</string></property>
  </action>
  <action name="action_Find">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/find.png</normaloff>:/main_ui/find.png</iconset></property>
   <property name="text"><string>&amp;Find</string></property>
   <property name="shortcut"><string>Ctrl+F</string></property>
  </action>
  <action name="action_Full_Screen">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/full_screen.png</normaloff>:/main_ui/full_screen.png</iconset></property>
   <property name="text"><string>&amp;Distraction Free</string></property>
   <property name="toolTip"><string>View full screen (distraction free)</string></property>
   <property name="shortcut"><string>F11</string></property>
  </action>
  <action name="action_Bug">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/report_bug.png</normaloff>:/main_ui/report_bug.png</iconset></property>
   <property name="text"><string>Report &amp;Bug/Request Feature</string></property>
  </action>
  <action name="action_Updates">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/download_cloud.png</normaloff>:/main_ui/download_cloud.png</iconset></property>
   <property name="text"><string>&amp;Check for Updates</string></property>
  </action>
  <action name="action_Cut">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/cut.png</normaloff>:/main_ui/cut.png</iconset></property>
   <property name="text"><string>Cut</string></property>
   <property name="shortcut"><string>Ctrl+X</string></property>
  </action>
  <action name="action_Copy">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/copy.png</normaloff>:/main_ui/copy.png</iconset></property>
   <property name="text"><string>&amp;Copy</string></property>
   <property name="shortcut"><string>Ctrl+C</string></property>
  </action>
  <action name="action_Paste">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/paste.png</normaloff>:/main_ui/paste.png</iconset></property>
   <property name="text"><string>Paste</string></property>
   <property name="shortcut"><string>Ctrl+V</string></property>
  </action>
  <action name="action_Select_All">
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/select_all.png</normaloff>:/main_ui/select_all.png</iconset></property>
   <property name="text"><string>Select All</string></property>
   <property name="shortcut"><string>Ctrl+A</string></property>
  </action>
  <action name="action_Wrap_Text">
   <property name="checkable"><bool>true</bool></property>
   <property name="icon"><iconset resource="main_ui.qrc"><normaloff>:/main_ui/wrap.png</normaloff>:/main_ui/wrap.png</iconset></property>
   <property name="text"><string>&amp;Wrap Text</string></property>
   <property name="toolTip"><string>Wrap long lines of text</string></property>
   <property name="shortcut"><string>Alt+W</string></property>
  </action>
 </widget>
 <resources>
  <include location="main_ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
